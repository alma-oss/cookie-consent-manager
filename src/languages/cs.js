export const config = {
  consent_modal: {
    title: 'Tyto stránky využívají cookies',
    description: `Kliknutím na „Přijmout vše“ dáváte souhlas společnosti LMC k využívání souborů Cookies a dalších identifikátorů ve vašem zařízení. Použití těchto Cookies a dalších identifikátorů usnadní navigaci na stránkách, zobrazení personalizovaného obsahu, cílený marketing, analýzu využívání našich produktů a služeb.
      Více informací naleznete na stránce <a href="https://www.lmc.eu/cs/cookies" target="_blank">Používání cookies</a>.`,
    primary_btn: {
      text: 'Přijmout vše',
      role: 'accept_all',
    },
    secondary_btn: {
      text: 'Přijmout nezbytné',
      role: 'accept_necessary',
    },
  },
  settings_modal: {
    blocks: [
      {
        toggle: {
          value: 'necessary',
          enabled: true,
          readonly: true,
        },
      },
      {
        toggle: {
          value: 'ad',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'analytics',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'functionality',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'personalization',
          enabled: false,
          readonly: false,
        },
      },
    ],
  },
};

export default config;
