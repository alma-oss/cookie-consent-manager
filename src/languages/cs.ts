import { addSeparators, pluralize } from '../utils';
import { CookieConsentCategory, ExtraMessages } from '../types';
import { VanillaCookieConsent } from '../types/vanilla-cookieconsent';

const extra = {
  and: 'a',
  company: 'společnosti',
  companies: 'společnostem',
};

/**
 * @param {ExtraMessages} [extraMessages] - Object with extra messages
 * @param {array} [extraMessages.companyNames] - Array of strings with company names used to parametrized translations
 * @returns {VanillaCookieConsent.Languages} Object with translated messages
 */
export const config = (extraMessages: ExtraMessages): VanillaCookieConsent.Languages => {
  const lang = { ...extra, ...extraMessages };

  return {
    consent_modal: {
      title: 'Díky Cookies budou naše stránky ještě lepší',
      description: `Když lépe pochopíme, co vás zajímá, budeme vám zobrazovat přesnější obsah na míru.
      Kliknutím na tlačítko „Přijmout všechny“ dáte ${pluralize(
        lang.companyNames.length,
        lang.company,
        lang.companies,
      )} ${addSeparators(
        lang.companyNames,
        lang.and,
      )} souhlas s využíváním souborů Cookies na účely personalizace, analýzy a cíleného marketingu.
      <a href="https://www.lmc.eu/cs/cookies" target="_blank">Co jsou to Cookies a jak je používáme?</a>`,
      primary_btn: {
        text: 'Přijmout všechny',
        role: VanillaCookieConsent.PrimaryButtonRole.ACCEPT_ALL,
      },
      secondary_btn: {
        text: 'Přijmout nezbytné',
        role: VanillaCookieConsent.SecondaryButtonRole.ACCEPT_NECESSARY,
      },
    },
    settings_modal: {
      blocks: [
        {
          toggle: {
            value: CookieConsentCategory.NECESSARY,
            enabled: true,
            readonly: true,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.AD,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.ANALYTICS,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.FUNCTIONALITY,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.PERSONALIZATION,
            enabled: false,
            readonly: false,
          },
        },
      ],
    },
  };
};

export default config;
