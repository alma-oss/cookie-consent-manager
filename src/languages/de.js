export const config = {
  consent_modal: {
    title: 'Diese Website verwendet Cookies',
    description: `Indem Sie auf „Alles akzeptieren“ klicken, stimmen Sie der Verwendung von Cookies und anderen Identifikatoren auf Ihrem Gerät durch LMC s.r.o. zu. Die Verwendung dieser Cookies und anderer Identifikatoren erleichtert die Navigation auf der Website, die Anzeige personalisierter Inhalte, gezieltes Marketing und die Analyse der Nutzung unserer Produkte und Dienstleistungen.
      Weitere Informationen finden Sie unter <a href="https://www.lmc.eu/en/cookies/" target="_blank">Verwendung von Cookies</a>.`,
    primary_btn: {
      text: 'Alles akzeptieren',
      role: 'accept_all',
    },
    secondary_btn: {
      text: 'Das Notwendigste akzeptieren',
      role: 'accept_necessary',
    },
  },
  settings_modal: {
    blocks: [
      {
        toggle: {
          value: 'necessary',
          enabled: true,
          readonly: true,
        },
      },
      {
        toggle: {
          value: 'ad',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'analytics',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'functionality',
          enabled: false,
          readonly: false,
        },
      },
      {
        toggle: {
          value: 'personalization',
          enabled: false,
          readonly: false,
        },
      },
    ],
  },
};

export default config;
