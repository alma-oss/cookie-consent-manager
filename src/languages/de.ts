import {
  addSeparators,
  assembleCategoryAd,
  assembleCategoryAnalytics,
  assembleCategoryFunctionality,
  assembleCategoryNecessary,
  assembleCategoryPersonalization,
  assembleDescriptionIntro,
  assembleSecondaryButton,
  isSettingsButtonNotShown,
  legalizeAlmaCareer,
} from '../utils';
import { CookieTableCategories, ExtraMessages, Values, VanillaCookieConsent } from '../types';
import { SecondaryButtonMode } from '../constants';

const extra = {
  and: 'und',
  legalName: 'Alma Career und seine Gruppenunternehmen',
};

/**
 * @param {ExtraMessages} [extraMessages] - Object with extra messages
 * @param {SecondaryButtonMode} [secondaryButtonMode] - Which secondary button should be shown
 * @param {CookieTableCategories} [cookieTable] - Cookie table items defined by category
 * @returns {VanillaCookieConsent.Languages} Object with translated messages
 */
export const config = (
  extraMessages: ExtraMessages,
  secondaryButtonMode: Values<typeof SecondaryButtonMode>,
  cookieTable: CookieTableCategories,
): VanillaCookieConsent.Languages => {
  const lang = { ...extra, ...extraMessages };

  return {
    consent_modal: {
      title: lang.consentTitle ?? 'Diese Website verwendet Cookies',
      description: `
      ${assembleDescriptionIntro(
        'Wenn wir genau wissen, wofür Sie sich interessieren, können wir Ihnen maßgeschneiderte Inhalte anbieten.',
        lang.descriptionIntro,
      )}
      <p>
        Indem Sie auf „Alles akzeptieren“ klicken, stimmen Sie der Verwendung von Cookies und anderen Identifikatoren auf Ihrem Gerät durch
        ${addSeparators(legalizeAlmaCareer(lang.companyNames, lang.legalName), extra.and)}
        zu. Die Verwendung dieser Cookies und anderer Identifikatoren erleichtert die Navigation auf der Website, die Anzeige personalisierter Inhalte, gezieltes Marketing und die Analyse der Nutzung unserer Produkte und Dienstleistungen.
        ${
          isSettingsButtonNotShown(secondaryButtonMode)
            ? `Sie können die Verwendung von Cookies in Ihren <strong><a href="" data-cc="c-settings">eigenen Einstellungen</a></strong> anpassen.`
            : ''
        }
      </p>`,
      primary_btn: {
        text: 'Alles akzeptieren',
        role: VanillaCookieConsent.PrimaryButtonRole.ACCEPT_ALL,
      },
      secondary_btn: assembleSecondaryButton(
        secondaryButtonMode,
        'Das Notwendigste akzeptieren',
        'Eigene Einstellungen',
      ),
    },
    settings_modal: {
      title: 'Benutzerdefinierte Cookie-Einstellungen',
      accept_all_btn: 'Alles akzeptieren',
      reject_all_btn: 'Das Notwendigste akzeptieren',
      save_settings_btn: 'Einstellungen speichern',
      cookie_table_headers: [
        { name: 'Unternehmensbezeichnung' },
        { description: 'Beschreibung' },
        { expiration: 'Verfallsdatum' },
      ],
      blocks: [
        {
          description:
            `Um unsere Website optimal nutzen zu können, sollten Sie alle Arten von Cookies aktivieren.\n` +
            (lang.settingsModalMoreInfo ??
              `Weitere Informationen darüber, was Cookies sind und wie wir mit ihnen arbeiten,
              finden Sie in unsere
              <a href="https://www.almacareer.com/gdpr" target="_blank">Datenschutzrichtlinien</a>.`),
        },
        assembleCategoryNecessary(
          'Technisch notwendige Cookies',
          'Diese Cookies sind für das reibungslose Funktionieren unserer Website unerlässlich und können daher nicht deaktiviert werden. Ohne sie könnten z. B. keine Inhalte auf unserer Seite angezeigt werden oder das Login würde nicht funktionieren.',
          cookieTable,
        ),
        assembleCategoryAnalytics(
          'Analytische Cookies',
          'Wir verwenden diese Cookies, um zu verfolgen, wie viele Personen unsere Website besuchen und wie sie sie nutzen. Auf diese Weise können wir die Website und andere Dienste kontinuierlich verbessern.',
          cookieTable,
        ),
        assembleCategoryFunctionality(
          'Funktionale Cookies',
          'Diese Cookies machen unsere Website leistungsfähiger und funktionieren besser. Sie ermöglichen uns zum Beispiel die Nutzung des Chats, damit wir Ihre Fragen schnell und einfach beantworten können.',
          cookieTable,
        ),
        assembleCategoryAd(
          'Marketing Cookies',
          'Mit diesen Cookies können wir messen, wie effektiv unsere Werbung und gezielte Angebote unserer Dienste sind. Marketing Cookies ermöglichen es uns, Sie online auf Nachrichten hinzuweisen, die für Sie von Interesse sein könnten.',
          cookieTable,
        ),
        assembleCategoryPersonalization(
          'Personalisierung Cookies',
          'Unsere Dienste funktionieren besser, wenn wir sie auf den einzelnen Nutzer zuschneiden können. Durch die Aktivierung von Personalisierungs-Cookies erhöhen Sie die Wahrscheinlichkeit, dass Sie genau die Inhalte finden, nach denen Sie suchen.',
          cookieTable,
        ),
      ],
    },
  };
};

export default config;
