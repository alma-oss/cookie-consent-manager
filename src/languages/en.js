import { addSeparators } from '../utils';

const extra = {
  and: 'and',
};

/**
 * @param {Object} [messages] - Object with extra messages
 * @param {array} [messages.companyNames] - Array of strings with company names used to parametrized translations
 * @returns {Object} Object with translated messages
 */
export const config = (messages) => {
  const lang = { ...extra, ...messages };

  return {
    consent_modal: {
      title: 'This website uses cookies',
      description: `By clicking on "Accept all", you give your consent to ${addSeparators(
        lang.companyNames,
        extra.and,
      )} to use cookies and other identifiers on your device. The use of these cookies and other identifiers will simplify navigation on the site, enable personalized content, targeted marketing, analysis of the usage of our products and services.
      For more information read page <a href="https://www.lmc.eu/en/cookies/" target="_blank">Use of cookies</a>.`,
      primary_btn: {
        text: 'Accept all',
        role: 'accept_all',
      },
      secondary_btn: {
        text: 'Accept necessary',
        role: 'accept_necessary',
      },
    },
    settings_modal: {
      blocks: [
        {
          toggle: {
            value: 'necessary',
            enabled: true,
            readonly: true,
          },
        },
        {
          toggle: {
            value: 'ad',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'analytics',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'functionality',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'personalization',
            enabled: false,
            readonly: false,
          },
        },
      ],
    },
  };
};

export default config;
