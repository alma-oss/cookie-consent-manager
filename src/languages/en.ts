import { addSeparators } from '../utils';
import { ExtraMessages, CookieConsentCategory } from '../types';
import { VanillaCookieConsent } from '../types/vanilla-cookieconsent';

const extra = {
  and: 'and',
};
/**
 * @param {ExtraMessages} [extraMessages] - Object with extra messages
 * @param {array} [extraMessages.companyNames] - Array of strings with company names used to parametrized translations
 * @returns {VanillaCookieConsent.Languages} Object with translated messages
 */
export const config = (extraMessages: ExtraMessages): VanillaCookieConsent.Languages => {
  const lang = { ...extra, ...extraMessages };

  return {
    consent_modal: {
      title: 'Cookies make our site even better',
      description: `By better understanding what you're interested in, we'll show you more relevant content.
      By clicking the "Accept all" button, you give ${addSeparators(
        lang.companyNames,
        extra.and,
      )} your consent to use cookies for personalisation, analytics and targeted marketing.
      <a href="https://www.lmc.eu/en/cookies/" target="_blank">What are cookies and how do we use them?</a>`,
      primary_btn: {
        text: 'Accept all',
        role: VanillaCookieConsent.PrimaryButtonRole.ACCEPT_ALL,
      },
      secondary_btn: {
        text: 'Accept necessary',
        role: VanillaCookieConsent.SecondaryButtonRole.ACCEPT_NECESSARY,
      },
    },
    settings_modal: {
      blocks: [
        {
          toggle: {
            value: CookieConsentCategory.NECESSARY,
            enabled: true,
            readonly: true,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.AD,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.ANALYTICS,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.FUNCTIONALITY,
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: CookieConsentCategory.PERSONALIZATION,
            enabled: false,
            readonly: false,
          },
        },
      ],
    },
  };
};

export default config;
