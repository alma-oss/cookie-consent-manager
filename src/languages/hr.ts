import {
  addSeparators,
  assembleDescriptionIntro,
  assembleSecondaryButton,
  isSettingsButtonNotShown,
  legalizeAlmaCareer,
  assembleCategoryNecessary,
  assembleCategoryAnalytics,
  assembleCategoryFunctionality,
  assembleCategoryAd,
  assembleCategoryPersonalization,
} from '../utils';
import { CookieTableCategories, ExtraMessages, Values, VanillaCookieConsent } from '../types';
import { CookieConsentCategory, SecondaryButtonMode } from '../constants';

const extra = {
  and: 'i',
  legalName: 'tvrtkama iz poslovne grupacije Alma Career',
};

/**
 * @param {ExtraMessages} [extraMessages] - Object with extra messages
 * @param {SecondaryButtonMode} [secondaryButtonMode] - Which secondary button should be shown
 * @param {CookieTableCategories} [cookieTable] - Cookie table items defined by category
 * @returns {VanillaCookieConsent.Languages} Object with translated messages
 */
export const config = (
  extraMessages: ExtraMessages,
  secondaryButtonMode: Values<typeof SecondaryButtonMode>,
  cookieTable: CookieTableCategories,
): VanillaCookieConsent.Languages => {
  const lang = { ...extra, ...extraMessages };

  return {
    consent_modal: {
      title: lang.consentTitle ?? 'Kolačići čine našu stranicu još boljom',
      description: `
      ${assembleDescriptionIntro(
        'Boljim razumijevanjem onoga što vas zanima, pokazat ćemo vam relevantniji sadržaj.',
        lang.descriptionIntro,
      )}
      <p>
        Klikom na gumb „Prihvati sve“, dajete
        ${addSeparators(legalizeAlmaCareer(lang.companyNames, lang.legalName), lang.and)}
        privolu za upotrebu kolačića za personalizaciju, analitiku i ciljani marketing..
        ${
          isSettingsButtonNotShown(secondaryButtonMode)
            ? `Možete prilagoditi upotrebu kolačića u svojim <strong><a href="" data-cc="c-settings">prilagođenim postavkama</a></strong>.`
            : ''
        }
      </p>`,
      primary_btn: {
        text: 'Prihvati sve',
        role: VanillaCookieConsent.PrimaryButtonRole.ACCEPT_ALL,
      },
      secondary_btn: assembleSecondaryButton(secondaryButtonMode, 'Prihvati nužno', 'Prilagođene postavke'),
    },
    settings_modal: {
      title: 'Prilagođene postavke kolačića',
      accept_all_btn: 'Prihvati sve',
      reject_all_btn: 'Prihvati nužno',
      save_settings_btn: 'Spremi postavke',
      cookie_table_headers: [{ name: 'Naziv' }, { description: 'Opis' }, { expiration: 'Valjanost' }],
      blocks: [
        {
          description:
            `Ako želite maksimalno iskoristiti našu web stranicu, najbolje je dopustiti sve vrste kolačića.\n` +
            (lang.settingsModalMoreInfo ??
              `Više informacija o tome što su kolačići i kako s njima radimo možete pronaći na stranici
              <a href="https://www.almacareer.com/gdpr" target="_blank">Pravila privatnosti</a>.`),
        },
        assembleCategoryNecessary(
          'Tehnički nužni kolačići',
          'Ovi kolačići su ključni za pravilno funkcioniranje naše web stranice i stoga ih nije moguće onemogućiti. Bez njih nije moguće prikazati sadržaj ili se prijaviti na našu web stranicu.',
          cookieTable,
        ),
        assembleCategoryAnalytics(
          'Analitički kolačići',
          'Ovi nam pomažu pratiti koliko ljudi posjećuje našu web stranicu i kako je koriste. Te informacije nam omogućuju kontinuirano poboljšavanje web stranice i drugih usluga.',
          cookieTable,
        ),
        assembleCategoryFunctionality(
          'Funkcionalni kolačići',
          'Naša web stranica djeluje još učinkovitije i bolje zahvaljujući ovim kolačićima. Na primjer, omogućuju nam korištenje usluge razgovora i brzo i jednostavno odgovaranje na vaša pitanja.',
          cookieTable,
        ),
        assembleCategoryAd(
          'Marketing kolačići',
          'Ovi kolačići nam pomažu mjeriti učinkovitost našeg oglašavanja i ciljanih ponuda usluga. Marketing kolačići omogućuju nam donošenje vijesti koje bi vas mogle zanimati na internetu.',
          cookieTable,
        ),
        assembleCategoryPersonalization(
          'Personalizacijski kolačići',
          'Naše usluge bolje funkcioniraju ako ih možemo prilagoditi određenim korisnicima. Dopuštanjem personalizacijskih kolačića povećavate šanse da pronađete sadržaj koji želite.',
          cookieTable,
        ),
      ],
    },
  };
};

export default config;
