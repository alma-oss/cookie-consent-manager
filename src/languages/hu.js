import { addSeparators } from '../utils';

const extra = {
  and: 'és',
};

/**
 * @param {Object} [messages] - Object with extra messages
 * @param {array} [args.companyNames] - Array of strings with company names used to parametrized translations
 * @returns {Object} Object with translated messages
 */
export const config = (messages) => {
  const lang = { ...extra, ...messages };

  return {
    consent_modal: {
      title: 'Az oldalak süti fájlokat használnak',
      description: `A „Mindent elfogadok” gombra kattintva a hozzájárulását adja ahhoz, hogy az ${addSeparators(
        lang.companyNames,
        extra.and,
      )} süti fájlokat és egyéb azonosítókat használjon az Ön eszközén. E süti fájlok és egyéb azonosítók használata megkönnyíti a weboldalon belüli navigációt, a személyre szabott tartalom megjelenítését, a célzott marketinget, valamint termékeink és szolgáltatásaink használatának elemzését.
      Bővebb információkat a <a href="https://www.lmc.eu/en/cookies/" target="_blank">Sütihasználat</a> oldalon talál.`,
      primary_btn: {
        text: 'Minden elfogadása',
        role: 'accept_all',
      },
      secondary_btn: {
        text: 'A legszükségesebbek elfogadása',
        role: 'accept_necessary',
      },
    },
    settings_modal: {
      blocks: [
        {
          toggle: {
            value: 'necessary',
            enabled: true,
            readonly: true,
          },
        },
        {
          toggle: {
            value: 'ad',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'analytics',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'functionality',
            enabled: false,
            readonly: false,
          },
        },
        {
          toggle: {
            value: 'personalization',
            enabled: false,
            readonly: false,
          },
        },
      ],
    },
  };
};

export default config;
