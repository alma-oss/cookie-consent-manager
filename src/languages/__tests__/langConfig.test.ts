import { config as configCs } from '../cs';
import { config as configDe } from '../de';
import { config as configEn } from '../en';
import { config as configHr } from '../hr';
import { config as configHu } from '../hu';
import { config as configPl } from '../pl';
import { config as configRu } from '../ru';
import { config as configSk } from '../sk';
import { config as configSl } from '../sl';
import { config as configUk } from '../uk';
import { SecondaryButtonMode } from '../../constants/SecondaryButtonMode';

describe.each([
  ['cs', configCs],
  ['de', configDe],
  ['en', configEn],
  ['hu', configHu],
  ['hr', configHr],
  ['pl', configPl],
  ['ru', configRu],
  ['sk', configSk],
  ['sl', configSl],
  ['uk', configUk],
])('config %s', (name, config) => {
  it('should return localization object for acceptNecessary button mode', () => {
    expect(
      config({ companyNames: ['test1', 'test2', 'test3'] }, SecondaryButtonMode.ACCEPT_NECESSARY),
    ).toMatchSnapshot();
  });

  it('should return localization object for showSettings button mode', () => {
    expect(config({ companyNames: ['test1', 'test2', 'test3'] }, SecondaryButtonMode.SHOW_SETTINGS)).toMatchSnapshot();
  });
});
