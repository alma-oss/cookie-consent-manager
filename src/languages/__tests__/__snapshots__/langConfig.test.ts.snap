// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config cs should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Když lépe pochopíme, co vás zajímá, budeme vám zobrazovat přesnější obsah na míru.
      Kliknutím na tlačítko „Přijmout všechny“ dáte společnostem test1, test2 a test3 souhlas s využíváním souborů Cookies na účely personalizace, analýzy a cíleného marketingu.
      <a href=\\"https://www.lmc.eu/cs/cookies\\" target=\\"_blank\\">Co jsou to Cookies a jak je používáme?</a>",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Přijmout všechny",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Přijmout nezbytné",
    },
    "title": "Díky Cookies budou naše stránky ještě lepší",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config de should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Indem Sie auf „Alles akzeptieren“ klicken, stimmen Sie der Verwendung von Cookies und anderen Identifikatoren auf Ihrem Gerät durch test1, test2 und test3 zu. Die Verwendung dieser Cookies und anderer Identifikatoren erleichtert die Navigation auf der Website, die Anzeige personalisierter Inhalte, gezieltes Marketing und die Analyse der Nutzung unserer Produkte und Dienstleistungen.
      Weitere Informationen finden Sie unter <a href=\\"https://www.lmc.eu/en/cookies/\\" target=\\"_blank\\">Verwendung von Cookies</a>.",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Alles akzeptieren",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Das Notwendigste akzeptieren",
    },
    "title": "Diese Website verwendet Cookies",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config en should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "By better understanding what you're interested in, we'll show you more relevant content.
      By clicking the \\"Accept all\\" button, you give test1, test2 and test3 your consent to use cookies for personalisation, analytics and targeted marketing.
      <a href=\\"https://www.lmc.eu/en/cookies/\\" target=\\"_blank\\">What are cookies and how do we use them?</a>",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Accept all",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Accept necessary",
    },
    "title": "Cookies make our site even better",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config hu should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "A „Mindent elfogadok” gombra kattintva a hozzájárulását adja ahhoz, hogy az test1, test2 és test3 süti fájlokat és egyéb azonosítókat használjon az Ön eszközén. E süti fájlok és egyéb azonosítók használata megkönnyíti a weboldalon belüli navigációt, a személyre szabott tartalom megjelenítését, a célzott marketinget, valamint termékeink és szolgáltatásaink használatának elemzését.
      Bővebb információkat a <a href=\\"https://www.lmc.eu/en/cookies/\\" target=\\"_blank\\">Sütihasználat</a> oldalon talál.",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Minden elfogadása",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "A legszükségesebbek elfogadása",
    },
    "title": "Az oldalak süti fájlokat használnak",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config pl should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Gdy lepiej zrozumiemy, co Cię interesuje, pokażemy dokładniejsze treści dopasowane do Twoich preferencji.
      Kliknij w przycisk „Akceptuj wszystkie”, aby wyrazić zgodę na wykorzystanie plików cookie przez
      firmy test1, test2 i test3 do personalizacji, analizy i ukierunkowanego marketingu.
      <a href=\\"https://www.lmc.eu/pl/cookies\\" target=\\"_blank\\">Co to są pliki cookie i jak je wykorzystujemy?</a>",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Akceptuj wszystkie",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Akceptuj niezbędne",
    },
    "title": "Dzięki plikom Cookies nasza strona będzie jeszcze lepsza",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config ru should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Нажав «Принять все», Вы даете свое согласие компаниям test1, test2 и test3 на использование файлов cookie и других идентификаторов на Вашем устройстве. Использование файлов cookie и других идентификаторов облегчит навигацию по сайту, отображения персонализированного контента, целевой маркетинг, анализ использования наших продуктов и услуг.
      Для получения дополнительной информации см. раздел <a href=\\"https://www.lmc.eu/en/cookies/\\" target=\\"_blank\\">использование файлов cookie</a>.",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Принять все",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Принятие необходимо",
    },
    "title": "Этот сайт использует файлы cookie",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config sk should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Presnejší obsah na mieru vám budeme zobrazovať, keď lepšie pochopíme, čo vás zaujíma.
      Kliknutím na tlačidlo „Prijať všetky“, čím dáte spoločnostiam test1, test2 a test3 súhlas s využívaním súborov Cookies za účelom personalizácie, analýzy a cieleného marketingu.
      <a href=\\"https://www.lmc.eu/sk/cookies\\" target=\\"_blank\\">Čo sú to Cookies a ako ich používame?</a>",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Prijať všetky",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Prijať nevyhnutné",
    },
    "title": "Vďaka Cookies budú naše stránky ešte lepšie",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;

exports[`config uk should return localization object 1`] = `
Object {
  "consent_modal": Object {
    "description": "Натиснувши «Прийняти все», Ви даєте свою згоду компаніям test1, test2 i test3 на використання файлів cookie та інших ідентифікаторів на Вашому пристрої. Використання цих файлів cookie та інших ідентифікаторів полегшить навігацію по сайту, відображення персоналізованого контенту, цільовий маркетинг, аналіз використання наших продуктів і послуг.
      Для отримання додаткової інформації див. розділ <a href=\\"https://www.lmc.eu/en/cookies/\\" target=\\"_blank\\">Використання файлів cookie</a>.",
    "primary_btn": Object {
      "role": "accept_all",
      "text": "Прийняти все",
    },
    "secondary_btn": Object {
      "role": "accept_necessary",
      "text": "Прийняття необхідно",
    },
    "title": "Цей сайт використовує файли cookie",
  },
  "settings_modal": Object {
    "blocks": Array [
      Object {
        "toggle": Object {
          "enabled": true,
          "readonly": true,
          "value": "necessary",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "ad",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "analytics",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "functionality",
        },
      },
      Object {
        "toggle": Object {
          "enabled": false,
          "readonly": false,
          "value": "personalization",
        },
      },
    ],
  },
}
`;
